app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -502,7 +503,9 @@ public void onBackPressed() {
             String toPage = UserPreferences.getDefaultPage();
             if (NavDrawerFragment.getLastNavFragment(this).equals(toPage)
                     || UserPreferences.DEFAULT_PAGE_REMEMBER.equals(toPage)) {
-                super.onBackPressed();
+                //super.onBackPressed();
+                // go to back ground, to quick restart this activity.
+                moveTaskToBack(false);
```

Comment:
This feels weird. It overwrites the default Android behavior of actually closing an app when pressing the back button. I don't think we should change that.